e2e/suite1/specs/dashboard-time-zone.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
+  describeName: 'Dashboard time zone support',
+  itName: 'Tests dashboard time zone scenarios',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    e2e.flows.openDashboard('5SdHCasdf');
+
+    const fromTimeZone = 'UTC';
+    const toTimeZone = 'America/Chicago';
+    const offset = -5;
+
+    const panelsToCheck = [
+      'Random walk series',
+      'Millisecond res x-axis and tooltip',
+      '2 yaxis and axis labels',
+      'Stacking value ontop of nulls',
+      'Null between points',
+      'Legend Table No Scroll Visible',
+    ];
+
+    const timesInUtc: Record<string, string> = {};
+
+    for (const title of panelsToCheck) {
+      e2e.components.Panels.Panel.containerByTitle(title)
+        .should('be.visible')
+        .within(() =>
+          e2e.components.Panels.Visualization.Graph.xAxis
+            .labels()
+            .should('be.visible')
+            .last()
+            .should(element => {
+              timesInUtc[title] = element.text();
+            })
+        );
+    }
+
+    e2e.pages.Dashboard.Toolbar.toolbarItems('Dashboard settings').click();
```

Comment:
nit: I just had an idea, this should really be this so we don't expose magic strings. But I can fix that in another PR.
```suggestion
    e2e.pages.Dashboard.Toolbar.dashboardSettings().click();
```

Reply:
ah, nice! I can update that one.